docs/dev/c4/containers.md

```diff
@@ -0,0 +1,37 @@
+```mermaid
+C4Context
+  title [Container] FloPy
+
+  Boundary(mf6, "MODFLOW 6"){
+    System(src, "Source code",)
```

Comment:
Consider adding a `Rel` from `src` to `dfn`.

Reply:
maybe we can get rid of the `src` system since it's an mf6 detail?

Reply:
Yes, let's just call one system MODFLOW6, and a SystemDb(dfn)